Dockerfile

```diff
@@ -2,26 +2,29 @@
 FROM ubuntu:14.04
 
 MAINTAINER IPython Project <ipython-dev@scipy.org>
- 
+
 # Make sure apt is up to date
 RUN apt-get update
 RUN apt-get upgrade -y
```

Comment:
I'll end up removing this line later, but feel free to modify it if I haven't merged yet. I'm running the build right now.